docker/updater/server.sh

```
@@ -0,0 +1,500 @@
+      project=$(docker inspect "$NOWLEDGE_MEM_CONTAINER" \
+                  --format '{{index .Config.Labels "com.docker.compose.project"}}' 2>/dev/null)
+      # Fallback when the container existed before compose v2's project
+      # labels (very old deploys) or was created by hand outside compose.
+      project_arg=""
+      [ -n "$project" ] && project_arg="-p $project"
+
+      # cd into the compose project dir so:
+      #   - relative overlay paths ("-f compose.updater.yaml") resolve here,
+      #     not against the sidecar's /opt/updater workdir;
+      #   - compose auto-discovers .env from the project dir.
+      # shellcheck disable=SC2086 — word-split $compose_overlays so
```

Shellcheck (0.11.0)

[error] Invalid key=value pair? Ignoring the rest of this directive starting here.

(SC1125)